server/tests/test-utils.ts

```diff
@@ -46,12 +49,10 @@ export const server = () => {
 };
 
 export const resetDb = async () => {
+  const config = getConfig();
   await createLocalTables(
-    { deleteTablesFirst: true },
-    {
-      region: 'eu-west-1',
-      endpoint: 'http://localhost:8000',
-    }
+    { deleteTablesFirst: true, tableNamePrefix: config.dynamoDBTablePrefix },
+    config.dynamoDB
```

Comment:
A thought here - should we actually make the setup (and therefore reset) isolated per test file so we can disable run-in-band?

Reply:
Will follow up in another PR